core/base-service/got.js

```diff
@@ -52,4 +55,6 @@ function fetchFactory(fetchLimitBytes = TEN_MB) {
   return sendRequest.bind(sendRequest, gotWithLimit)
 }
 
-export { fetchFactory, userAgent }
+const fetch = _fetchFactory()
+
+export { fetch, _fetchFactory }
```

Comment:
This is not my favourite solution ever. Testing this with different values for `fetchLimitBytes` is tricky, but I do prefer the ergonomics of exporting `fetch` "fully formed" as it were. Exporting an underscore prefixed function we can test from the module seems like a reasonable tradeoff but I'm not wedded to it.. we could go back.

Reply:
> Exporting an underscore prefixed function we can test from the module seems like a reasonable tradeoff but I'm not wedded to it.. we could go back.

We do the same in plenty of places elsewhere so I've no concerns with it